sf-docker/multi-camera/docker-compose.yml

```diff
@@ -274,6 +294,44 @@ services:
     # - "/var/tmp/innovatrics/tensor-rt:/var/tmp/innovatrics/tensor-rt"
     #runtime: nvidia
 
+  pedestrian-extractor:
```

Comment:
this service is duplicated everywhere it was added

Reply:
sorry removed